shared/datatypes.yaml

```diff
@@ -17,20 +17,25 @@ fastq.gz: fastqsanger.gz
 fastq.bz: fastqsanger.bz2
 fastqsanger: fastqsanger
 fastqsanger.gz: fastqsanger.gz
-fastqsanger.bz: fastqsanger.bz2
-fna: fasta
+fastqsanger.bz: fastqsanger.bz
 fa: fasta
+fb: tabular
```

Comment:
@bedroesb do you know form where this is coming? what is `fb`? Is that just a typo?

Reply:
The `.fb` file is found in the goenrichment tutorial in transcriptomics. See [zenodo](https://zenodo.org/record/2565417)
While searching for the extension I found some data-library files that were named incorrectly (see latest push).

Reply:
I see this refers to FlyBase probably. I don't think we should add this here instead we should fix this by hand and ask @dsobral to change this to tabular the next time he changes the data.

Reply:
I delete the .fb extention?

Reply:
Yes, actually the file is a gaf file, it is not even tabular...
At some point it was renamed and the file format was "lost in translation".

Sorry, I need to change it to .gaf  
This will also imply changing a few things at different points in the process.

Answering @bedroesb the .fb extension, at least in what regards this dataset, should not be included.

Best,
Daniel

Reply:
ok thank you. Done